Formula/arduino-cli.rb

```diff
@@ -0,0 +1,22 @@
+class ArduinoCli < Formula
+  desc "Arduino command-line interface"
+  homepage "https://github.com/arduino/arduino-cli"
+  url "https://github.com/arduino/arduino-cli/archive/0.3.2-alpha.preview.tar.gz"
+  sha256 "fe0a2766dbe9410e7b9873778c0fe7e693cb82e29db1ce246e9a7eeff2569ed7"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/arduino/arduino-cli").install buildpath.children
+
+    cd "src/github.com/arduino/arduino-cli" do
+      system "go", "build", "-o", bin/"arduino-cli"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system bin/"arduino-cli", "version"
```

Comment:
Can we test more functionality of the app? From the [docs](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula):

> We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. However, a bad test is better than no test at all.

Reply:
Yes, but they will require installation of more components (Board packages). I can do that, but I'm not sure that's what you want.

Reply:
@igas Hello?

Reply:
Sorry for the delay. I can see in readme:
```
$ arduino-cli sketch new MyFirstSketch
Sketch created in: /home/luca/Arduino/MyFirstSketch

$ cat /home/luca/Arduino/MyFirstSketch/MyFirstSketch.ino
void setup() {
}

void loop() {
}
```
We can do the same, assert that `sketch new` have output `Sketch created` and that `MyFirstSketch/MyFirstSketch.ino` equal this content.